CHANGELOG.md

```diff
@@ -50,27 +49,6 @@ New features, fixed bugs, known defects and other noteworthy changes to each rel
 ### Changed
 - Bumped version to 2.3.0 for helm charts to match with backend release.
 
-## [2.1.4] -  non-released
-### Fixed
-- Data table pagination not working - fixed
```

Comment:
So you fixed it, removed it because you didn't do a release but never added it to another section? Not sure if i understnad your change

Reply:
Hi @Siegfriedk , We have corrected it here #78 

Thanks